test/api/assessment/samples.go

```diff
@@ -17,28 +17,28 @@ var (
 		Questionnaire: api.Ref{
 			Name: questionnaire.Questionnaire1.Name,
 		},
-		Sections: []assessment.Section{
+		Sections: []api.Section{
```

Comment:
Package discrepancies for nested structs are because `api.Section` is defined as `model.Section`, so all the nested structs are from the `model` package.